addon/components/ember-tutorial-component.js

```diff
@@ -167,17 +179,37 @@ export default Ember.Component.extend({
    * @return `left: Xpx; top: Ypx;`
    */
   tooltipStyle:  computed('xCoord', 'yCoord', function() {
-    return new SafeString(`left: ${this.get('xCoord') + 15}px; top: ${this.get('yCoord')}px; position: absolute;`);
+    return new SafeString(`left: ${this.get('xCoord')}px; top: ${this.get('yCoord')}px; position: absolute;`);
   }),
 
   /**
    * Initialize the tooltip
-   * @method initTootltip
+   * @method initTooltip
    */
-  initTootltip() {
+  initTooltip() {
     this.set('currentConfigIndex', 0);
-    this.computeXCord();
-    this.computeYCord();
+    this.setPosition();
+  },
+
+  /**
+   * Overrides the init function
+   */
+  init() {
+    this._super(...arguments);
+
+    Ember.$(window).on('resize.tutorial-component', function() {
+      console.log('resize position and trigger setPosition');
+      this.setPosition();
```

Comment:
Thanks for adding this, wanted to add that in the initial release itself but missed it.